src/TLuaInterpreter.cpp

```diff
@@ -74,16 +75,28 @@
 #include "luazip.h"
 #endif
 
+const QMap<Qt::MouseButton, QString> TLuaInterpreter::mMouseButtons = {
+        {Qt::NoButton, QStringLiteral("NoButton")},           {Qt::LeftButton, QStringLiteral("LeftButton")},       {Qt::RightButton, QStringLiteral("RightButton")},
```

Comment:
🤔 @vadi2 I can't recall but is `QLatin1String` better or worse than `QStringLiteral` here?  If there is `%1` type parameter replacement or non-ASCII characters it has to be the `QStringLiteral`type which are stored in the read-only part of the executable and the construction cost is nil at _run-time_ but each instance in the source has to be stored separately (no sharing of identical strings) but what are the features of `QLatin1String` in comparision?

Reply:
Let's go with `QStringLiteral` and not worry about it. The impact will not be significant between these two here I don't think.